pkg/backend/kubeflow/generator/cm.go

```diff
@@ -0,0 +1,213 @@
+package generator
```

Comment:
what happens to the `cm`, should we also have `destroyer`?

I think it's a general question about how to clean up resource created from ciao.

Reply:
Ref #13 

I think we should have such a thing to delete all resources after the job is finished.

Reply:
But maybe not in this PR.